jumpcloud-ADMU/Powershell/Private/DisplayForms/Form.ps1

```diff
@@ -713,6 +792,17 @@ Function Show-SelectionForm {
                     Write-ToLog "User $($tb_JumpCloudUserName.Text) does not have a local account on this system"
                 }
             }
+            if ([string]::IsNullOrEmpty($SelectedUserName)) {
+                $SelectedUserName = $($lvProfileList.SelectedItem.username)
+            }
+
+            # Preload progress form to shrink perceived wait between click and display
+            $profilePath = $lvProfileList.SelectedItem.LocalPath
+            $displayUsername = if ([string]::IsNullOrWhiteSpace($tb_JumpCloudUserName.Text)) { $SelectedUserName } else { $tb_JumpCloudUserName.Text }
+            $script:ProgressBar = New-ProgressForm
+            $FormResults.ProgressBar = $script:ProgressBar
+            Write-ToProgress -form $true -ProgressBar $script:ProgressBar -status "Init" -username $SelectedUserName -newLocalUsername $displayUsername -profileSize "Calculating" -LocalPath $profilePath
```

Comment:
We should be able to get the profileSize:

`$profileSize = Get-ProfileSize -ProfilePath $localPath $lvProfileList.SelectedItem.LocalPath`

Reply:
I guess that makes sense. I was thinking of calculating it asynchronously but that would add complexity.

Reply:
Yeah I'm game to keep it as is, I want the form to load as quick as possible, it's fine if it says calculating for a bit. IMO this is a change admins will appreciate.